app/controllers/credentials/worldpay.controller.js

```diff
@@ -35,12 +35,26 @@ async function updateWorldpayCredentials (req, res, next) {
     const credential = getCredentialByExternalId(req.account, req.params.credentialId)
     const results = credentialsForm.validate(req.body)
 
+    if (!results.errorSummaryList.length) {
+      const merchantId = req.body.merchantId
+      if (req.account.allow_moto && !merchantId.endsWith('MOTO')) {
+        results.errors.merchantId = 'Enter a valid MOTO merchant account code. MOTO payments are enabled for the account'
+      } else if (!req.account.allow_moto && merchantId.endsWith('MOTO')) {
+        results.errors.merchantId = 'Enter a valid merchant account code. Please contact support if you would like moto payments enabled'
```

Comment:
The acronym ‘moto’ should definitely be capitalised. But maybe this message could more direct? Something like “MOTO merchant code not allowed. Please contact support if you would like MOTO payments enabled”.